src/components/DayOverview/DayOverviewButton.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
```

Comment:
This is looking good! My only comment would be is it possible for you to make this component look something like `src\components\ShippingLabels\ShippingLabelsButton.tsx` where there are no data logic in the file at all. So things like date and location should be passed in from inside the data props.

There's absolutely nothing wrong with the current implementation. Its just this file will be changed to a generic PDF button component which is implemented in VFB-11, and your current implementation will not be able to use that directly.

Reply:
Done